scripts/export_environment.py

```diff
@@ -250,23 +259,19 @@ def resolve_env(env: Environment) -> Environment:
 
     {'x': '42', 'y': '42', 'o': '{', 'c': '}'}
 
-    Dangling braces are not supported:
+    JSON objects are not resolved, but can be stored as JSON strings.
 
-    >>> resolve_env({'x': '{'})
-    Traceback (most recent call last):
-    ...
-    ValueError: Single '{' encountered in format string
-    >>> resolve_env({'x': '}'})
-    Traceback (most recent call last):
-    ...
-    ValueError: Single '}' encountered in format string
+    >>> resolve_env({'x': '{}'})
+    {'x': '{}'}
 
-    And an empty pair of braces isn't either:
+    >>> resolve_env({'x':'{ }'})
+    {'x': '{ }'}
 
-    >>> resolve_env({'x': '{}'})
-    Traceback (most recent call last):
-    ...
-    ValueError: Format string contains positional fields
+    >>> r = resolve_env({'x': '{"foo": "bar"}'})
+    >>> r
+    {'x': '{"foo": "bar"}'}
+    >>> r['x']
+    '{"foo": "bar"}'
```

Comment:
The test for `r['x']` seems unnecessary since the full value of `r` is already being tested.

Reply:
Good point, I'll change block.